avm/res/web/serverfarm/main.bicep

```diff
@@ -136,16 +136,20 @@ resource avmTelemetry 'Microsoft.Resources/deployments@2024-03-01' = if (enableT
   }
 }
 
-resource appServicePlan 'Microsoft.Web/serverfarms@2022-09-01' = {
+resource appServicePlan 'Microsoft.Web/serverfarms@2024-11-01' = {
```

Comment:
I am guessing this will break some of the properties. Since there are some breaking changes in the version it seems.